config/routes.rb

```diff
@@ -14,12 +14,17 @@
     get 'rebate_forms/signed', to: 'signed_rebate_forms#index'
     get 'rebate_forms/signed/download_csv', to: 'signed_rebate_forms#download_csv'
     get 'rebate_forms/processed', to: 'processed_rebate_forms#index'
+    get 'rebate_forms/declined', to: 'declined_rebate_forms#index'
     post 'process_rebate_form', to: 'processed_rebate_forms#create'
     delete 'unprocess_rebate_form', to: 'processed_rebate_forms#destroy'
     delete 'unprocess_rebate_forms', to: 'processed_rebate_forms#destroy_all'
 
     resources :rebate_forms do
       get 'generateqr'
```

Comment:
Is :arrow_down: how you'd prefer to do it or is it a work around? Would it make more sense to use the request builder to directly submit `patch` requests rather than routing through  a get?

Reply:
`get 'rebate_forms/declined'` is to get the list of declined forms for the new tab, you might be thinking of something else.